src/bigrand.rs

```diff
@@ -14,6 +14,9 @@ use bigint::{into_magnitude, magnitude};
 use integer::Integer;
 use traits::Zero;
 
+/// A trait for sampling random big integers.
+///
+/// The `rand` feature must be enabled to use this. See crate-level documentation for details.
```

Comment:
Do you mean to reference `README.md`? The crate-level documentation would be at the top of `lib.rs`, which doesn't mention features at all. (But it maybe should!)

Reply:
Whoops! Good catch.

I'm not sure if there's an existing convention, but it seems reasonable to have features described in both places.